viewing_party/party.py

```diff
@@ -0,0 +1,201 @@
+
+#this is the function that creates a new movie
+def create_movie(title, genre, rating):
+
```

Comment:
I highly suggest adding docstrings underneath function signatures. The docstrings are a neat way of adding multi-line comments, especially about the functions we are building. It's also a great way of providing notes to your future self.

This same advice can be applied to the rest of the functions in this project and future projects.
```suggestion
def create_movie(title, genre, rating):
    '''
    this is the function that creates a new movie
    '''
```